File: 1878-check-if-array-is-sorted-and-rotated/1878-check-if-array-is-sorted-and-rotated.go
func check(nums []int) bool {
    cnt := 0

    for i := 0; i < len(nums); i++ {
        if nums[i] > nums[(i+1) % len(nums)] {
            cnt += 1
        }
        if cnt > 1 {
            return false
        }
    }  
    return true
}

File: 1878-check-if-array-is-sorted-and-rotated/1878-check-if-array-is-sorted-and-rotated.rs
impl Solution {
    pub fn check(nums: Vec<i32>) -> bool {
        let mut cnt: i16 = 0;
        for i in 0..nums.len() {
            if nums[i] > nums[(i + 1) % nums.len()] {
                cnt += 1;
            }
            if cnt > 1 {
                return false;
            }
        }  
        return true;
    }
}